apps/frontend/static_src/js/main.js

```diff
@@ -1 +1,63 @@
 import { Dropdown } from 'bootstrap';
+require('./lodash_debounce/lodash.custom.min.js')
```

Comment:
I'm a bit surprised we're adding this manually 🤔 

Can't we install it through NPM and `import` it?

Reply:
Ohh we can. We only needed the `debounce` function of lodash, so I thought installing the whole `lodash` package would not be necessary and hence I included the custom build which enables us to use debounce.
However it turns out there is also a npm package which allows us to do the same https://www.npmjs.com/package/lodash.debounce
I will change it to use npm package.